man/io_uring_setup.2

```diff
@@ -148,6 +149,27 @@ The
 .I resv
 array must be initialized to zero.
 
+.I features
+is filled in by the kernel, which specifies various features supported
+by current kernel version.
+.TP
+.B IORING_FEAT_SINGLE_MMAP
+.TP
+.B IORING_FEAT_NODROP
```

Comment:
You should add here:

If this flag is set, io_uring supports never dropping completion events. If a completion event occurs and the CQ ring is full, the kernel stores the event internally until such a time that the CQ ring has room for more entries. If this overflow condition is entered, attempting to submit more IO with fail with the
.B -EBUSY
error value, if it can't flush the overflown events to the CQ ring. If this happens, the application must reap events from the CQ ring and attempt the submit again.